tests/testthat/_snaps/chop.md

```diff
@@ -4,29 +4,29 @@
       (expect_error(chop(df)))
     Output
       <error/rlang_error>
-      Argument `cols` is missing with no default
+      Error in `check_present()`: Argument `cols` is missing with no default
 
 # the ptype must be a list
 
     Code
       (expect_error(unchop(mtcars, mpg, ptype = 1)))
     Output
       <error/rlang_error>
-      `ptype` must be a named list.
+      Error in `df_unchop()`: `ptype` must be a named list.
 
 # incompatible sizes are caught
 
     Code
       (expect_error(unchop(df, c(x, y))))
     Output
       <error/rlang_error>
-      In row 1, can't recycle input of size 2 to size 3.
+      Error in `fn()`: In row 1, can't recycle input of size 2 to size 3.
```

Comment:
This is unfortunate, it comes from using `purrr::reduce()` so I think it is out of our control

Reply:
We could probably pass a lambda to avoid this.